Assets/MixedRealityToolkit/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+
+namespace Microsoft.MixedReality.Toolkit.InputSystem
+{
+    /// <summary>
+    /// The Boundary system controls the presentation and display of the users boundary in a scene
+    /// </summary>
+    public class MixedRealityBoundaryManager : BaseManager, IMixedRealityBoundarySystem
+    {
+        /// <summary>
+        /// MixedRealityBoundaryManager constructor
+        /// </summary>
+        public MixedRealityBoundaryManager()
+        {
+            //Attach to Manager events
+            MixedRealityManager.Instance.InitializeEvent += InitializeInternal;
+            MixedRealityManager.Instance.UpdateEvent += Update;
+            MixedRealityManager.Instance.ProfileUpdateEvent += ProfileUpdate;
+            MixedRealityManager.Instance.DestroyEvent += Destroy;
```

Comment:
If we're using interfaces for the managers, we don't need to subscribe, we can directly call the methods in each child manager.

Reply:
OK, that's why I put up both choices for us to determine.  The subscribers in children makes more sense as each manager can choose what to subscribe to, however, having it in the base class is cleaner :D 
Right, that's the path then

Reply:
Agreed with your new method